pkg/fileservice/checks/interval/interval_tree.go

```diff
@@ -0,0 +1,964 @@
+// Copyright 2016 The etcd Authors
```

Comment:
Is the Copyright Notice in correct format?

Reply:
Used similar style from other parts of the code: https://github.com/matrixorigin/matrixone/blob/64cda0138af5af0d40c47ebc9423298e1e7fef28/pkg/util/trace/type.go#L1